### PR TITLE
bugfix/type-not-found-rejection

### PR DESCRIPTION
**Pull request by wkrugiolka**
#### Todo

- Tests
    - [ ] Unit
    - [ ] Integration
    - [ ] Acceptance
- [ ] Documentation

### CODE REVIEW

File: src/coffee/stockimport.coffee [hunk 1]
```diff
@@ -211,6 +211,8 @@ class StockImport
             @_getCustomTypeDefinition(customTypeKey).then (response) =>
               customTypeDefinition = response.body
               @_mapCustomField(_data, cell, headerName, customTypeDefinition, rowIndex)
+            .catch (error) =>
+              return reject(error)
```
Comment (daern91) [inline src/coffee/stockimport.coffee]: Why catch to re-throw here?

Comment (daern91) [reply src/coffee/stockimport.coffee]: Ah, nevermind, its for the outside promise. I see.

**Comment by wkrugiolka** [comment]
Exactly :)
 
I see that you merged it, please let me know when you release new version of
this lib.
 
 
Regards, Wojtek

**Comment by daern91** [comment]
Absolutely! Done now as `v3.0.3` https://www.npmjs.com/package/sphere-stock-import/v/3.0.3

**Comment by wkrugiolka** [comment]
Thank you

**Pull request merged**